### PR TITLE
added repository readme

### PR DESCRIPTION
**Pull request by cnsr**

### CODE REVIEW

**Comment by cnsr** [comment]
@erikh360 please, look at this readme and add anything you feel should be included

File: README.md [hunk 1]
```diff
@@ -2,11 +2,59 @@
...
+
+## Submitting a PR
+Before submitting a PR make sure you have ran tests
+```sh
 $ pip install -e .
 $ python manage.py test
 ```
+and formatted your code. To do so, run:
+```
+$ isort **/*.py
```
Comment (erikh360) [inline README.md:61]: We could also add these to precommit hook.

**Review by erikh360 — APPROVED** [review]
Looks good just one suggestion but not a blocker

File: README.md (hunk 1 above)
Comment (cnsr) [reply README.md:61]: We aren't using precommit in this repository but i would consider adding it

**Comment by cnsr** [comment]
I've added some more descriptive READMEs to each of the applications. Not sure if neccessary but could be helpful if they are to be worked on by someone new.

**Pull request merged**